apps/api/src/utils/mailjet.js

```diff
@@ -0,0 +1,40 @@
+const config = require("../config");
+
+const mailjet = require("node-mailjet").connect(
+  config.mailjet.publicKey,
+  config.mailjet.privateKey
+);
+
+const sendRequestConfirmation = (emailTo, nameTo, maskRequestId) => {
+  const request = mailjet.post("send", { version: "v3.1" }).request({
+    Messages: [
+      {
+        From: {
+          Email: "contact@usinepartagee.fr",
+          Name: "Usine partagée Normandie"
+        },
+        To: [
+          {
+            Email: emailTo,
+            Name: nameTo
+          }
+        ],
+        TemplateID: 1323123,
```

Comment:
Maybe it's a good idea to put the template ID in an environment variable ?

Reply:
done!